projects/demo-playwright/tests/legacy/input-date-time/input-date-time.pw.spec.ts

```diff
@@ -310,4 +310,37 @@ test.describe('InputDateTime', () => {
             });
         });
     });
+
+    test.describe('Examples', () => {
+        let example!: Locator;
+        let documentationPage!: TuiDocumentationPagePO;
+        let inputDateTime!: TuiInputDateTimePO;
+
+        test.beforeEach(async ({page}) => {
+            await tuiGoto(page, DemoRoute.InputDateTime);
+
+            documentationPage = new TuiDocumentationPagePO(page);
+            example = documentationPage.apiPageExample;
+
+            inputDateTime = new TuiInputDateTimePO(
+                example.locator('tui-input-date-time'),
+            );
+        });
+
+        test('With validator: enter incomplete date -> validator error', async () => {
+            example = documentationPage.getExample('#with-validator');
+            inputDateTime = new TuiInputDateTimePO(
+                example.locator('tui-input-date-time'),
+            );
+
+            await inputDateTime.textfield.clear();
+            await inputDateTime.textfield.fill('11');
+            await inputDateTime.textfield.blur();
+
+            await expect(example).toHaveScreenshot(
+                '04-input-data-time-with-validator.png',
+                {animations: 'allow'},
```

Comment:
Should we add some wait before this screenshot for test stability ?

```suggestion
            await page.waitForTimeout(100);
            
            await expect(example).toHaveScreenshot(
                '04-input-data-time-with-validator.png',
                {animations: 'allow'},
```

Reply:
I do not know, is it really necessary? It is still working